### ISSUE TITLE
error "Requested device not found"

### ISSUE THREAD

**Issue by bipin1611** [westonsoftware/vue-webrtc#76]
First of all, Thank you for the Great work!

I have installed it on my project and added it at Two Places. At One Place it's working fine, but in Other Place, it's giving me the "Requested device not found" error, even if I have the same network, same browser,

Please help here.

Thank you!

**Comment by westonsoftware** [comment]
Hi, check your deviceId ... set it to null for the default camera or let your user pick one.

**Comment by bipinparmar-ss** [comment]
Yes, `navigator.mediaDevices.getUserMedia` are not getting deviceId in response.

As, I am very new to this, how do we get deviceId or generate new deviceId each time when this API call.?

Appreciate your help!

Thanks you!
